RNTester/RNTester.xcodeproj/project.pbxproj

```diff
@@ -1707,19 +1707,19 @@
 			shellPath = /bin/sh;
 			shellScript = "export NODE_BINARY=node\n$SRCROOT/../scripts/react-native-xcode.sh RNTester/js/RNTesterApp.ios.js";
 		};
-		68CD48B71D2BCB2C007E06A9 /* Run Script */ = {
+		68CD48B71D2BCB2C007E06A9 /* Bundle React Native code and images */ = {
 			isa = PBXShellScriptBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
 			);
 			inputPaths = (
 			);
-			name = "Run Script";
+			name = "Bundle React Native code and images";
 			outputPaths = (
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export NODE_BINARY=node\n$SRCROOT/../scripts/react-native-xcode.sh RNTester/js/RNTesterApp.ios.js\n";
+			shellScript = "export NODE_BINARY=node\n$SRCROOT/../scripts/react-native-xcode.sh RNTester/js/RNTesterApp.ios.js $SRCROOT/..\n";
```

Comment:
```suggestion
			shellScript = "export NODE_BINARY=node\nPROJECT_ROOT=$SRCROOT/.. $SRCROOT/../scripts/react-native-xcode.sh RNTester/js/RNTesterApp.ios.js\n";
```

Reply:
Would this work, in place of adding a second parameter to the script? It would remove the need for $PROJECT_ROOT_OVERRIDE.

Reply:
Supplying PROJECT_ROOT via env var whilst starting the `react-native-xcode.sh` won't override the PROJECT_ROOT var inside the script, as the var is constructed inside the shell script (with no existing logic to accept overrides externally).

https://github.com/facebook/react-native/blob/fb8a7c3cc51f2b68f5d82266ea5f6525ab396310/scripts/react-native-xcode.sh#L55-L60

I agree with you that supplying the PROJECT_ROOT_OVERRIDE via script arg is not as elegant as supplying it via env var. In the case of the latter approach, the initialization of the script will look something like:

```bash
PROJECT_ROOT_OVERRIDE=$SRCROOT/.. $SRCROOT/../scripts/react-native-xcode.sh RNTester/js/RNTesterApp.ios.js;
```

If this is workable, i'll go ahead with it 👍

Reply:
My understanding is it should be overridable. The following line sets PROJECT_ROOT to the current value of PROJECT_ROOT, unless it's nil, in which case it is set to "$REACT_NATIVE_DIR/../..".

```
PROJECT_ROOT=${PROJECT_ROOT:-"$REACT_NATIVE_DIR/../.."} 
```

I verified that's the case in https://circleci.com/gh/hramos/react-native/5739
. I'll apply those changes and land.